File: Main/ReferenceDLLs/Pack1/Facebook Visual Kit CS/Demo Application/FVK_Demo/Components2Master.Master.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace FVK_Demo
{
    public partial class Components2Master : System.Web.UI.MasterPage
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            if (!Page.IsPostBack)
            {
                tab3.AddTab("Invite Friends", "Invite.aspx", 120);
                tab3.AddTab("Invite F. Small", "InviteSmall.aspx", 120);
                tab3.AddTab("Comments", "Comments.aspx", 120);
                tab3.AddTab("Select Friends", "SelectFriends.aspx", 120);
                tab3.AddTab("Send Status", "SendStatus.aspx", 120);

                tab3.Distance = 5;
                tab3.BorderWidth = 10;
            }
        }
    }
}

File: Main/ReferenceDLLs/Pack1/Facebook Visual Kit VB/Demo Application/FVK_Demo_VB/Components2Master.Master.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls

Namespace FVK_Demo
    Partial Public Class Components2Master
        Inherits System.Web.UI.MasterPage
        Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
            If Not Page.IsPostBack Then
                tab3.AddTab("Invite Friends", "Invite.aspx", 120)
                tab3.AddTab("Invite F. Small", "InviteSmall.aspx", 120)
                tab3.AddTab("Comments", "Comments.aspx", 120)
                tab3.AddTab("Select Friends", "SelectFriends.aspx", 120)
                tab3.AddTab("Send Status", "SendStatus.aspx", 120)

                tab3.Distance = 5
                tab3.BorderWidth = 10
            End If
        End Sub
    End Class
End Namespace
